mobile_codes/source_data/itu.json

```diff
@@ -1,1745 +1,2038 @@
-    ["746", "04", "Intelsur", "Intelsur"],
-    ["746", "05", "Telesur (CDMA)", "Telesur (CDMA)"],
-    ["748", "00", "Ancel - TDMA", "Ancel - TDMA"],
-    ["748", "01", "Ancel - GSM", "Ancel - GSM"],
-    ["748", "03", "Ancel", "Ancel"],
-    ["748", "07", "Movistar", "Movistar"],
-    ["748", "10", "CTI M\u00f3vil", "CTI M\u00f3vil"],
-    ["750", "001", "Touch", "Touch"],
-    ["901", "01", "ICO Global Communications", "ICO Global Communications"],
-    ["901", "03", "Iridium Communications Inc", "Iridium Communications Inc"],
-    ["901", "05", "Thuraya RMSS Network", "Thuraya RMSS Network"],
-    ["901", "06", "Thuraya Satellite Telecommunications Company", "Thuraya Satellite Telecommunications Company"],
-    ["901", "10", "Asia Cellular Satellite (AceS)", "Asia Cellular Satellite (AceS)"],
-    ["901", "11", "Inmarsat Ltd.", "Inmarsat Ltd."],
-    ["901", "12", "Maritime Communications Partner AS (MCP network)", "Maritime Communications Partner AS (MCP network)"],
-    ["901", "13", "BebbiCell AG (Formerly Global Networks Switzerland AG)", "BebbiCell AG (Formerly Global Networks Switzerland AG)"],
-    ["901", "14", "Telenor", "Telenor"],
-    ["901", "15", "OnAir N.V. (Formerly SITA on behalf of Onair)", "OnAir N.V. (Formerly SITA on behalf of Onair)"],
-    ["901", "16", "Jasper Wireless, Inc", "Jasper Wireless, Inc"],
-    ["901", "17", "Jersey Telecom", "Jersey Telecom"],
-    ["901", "18", "Cingular Wireless", "Cingular Wireless"],
-    ["901", "19", "Vodafone Malta (Vodafone Group)", "Vodafone Malta (Vodafone Group)"],
-    ["901", "20", "Intermatica", "Intermatica"],
-    ["901", "21", "Wins Limited (Formerly Seanet Maritime Communications AB)", "Wins Limited (Formerly Seanet Maritime Communications AB)"],
-    ["901", "22", "MediaLincc Ltd", "MediaLincc Ltd"],
-    ["901", "24", "Voxbone SA", "Voxbone SA"],
-    ["901", "26", "Telecom Italia", "Telecom Italia"],
-    ["901", "27", "OnAir N.V. (Formerly SITA on behalf of Onair)", "OnAir N.V. (Formerly SITA on behalf of Onair)"],
-    ["901", "28", "Vodafone Group", "Vodafone Group"],
-    ["901", "29", "Telenor Connexion AB", "Telenor Connexion AB"],
-    ["901", "31", "France Telecom Orange", "France Telecom Orange"],
-    ["901", "32", "MegaFon", "MegaFon"],
-    ["901", "33", "Smart Communications , Inc", "Smart Communications , Inc"],
-    ["901", "34", "Tyntec GmbH", "Tyntec GmbH"],
-    ["901", "35", "Globecomm Network Services", "Globecomm Network Services"],
-    ["901", "36", "Azerfon LLC", "Azerfon LLC"],
-    ["901", "37", "TRANSATEL", "TRANSATEL"],
-    ["901", "38", "Multiregional TransitTelecom (MTT)", "Multiregional TransitTelecom (MTT)"],
-    ["901", "39", "MTX Connect", "MTX Connect"],
-    ["901", "40", "Deutsche Telekom AG", "Deutsche Telekom AG"],
-    ["901", "88", "UN Office for the Coordination of Humanitarian Affairs (OCHA)", "UN Office for the Coordination of Humanitarian Affairs (OCHA)"]
+    ["202", "01", "Cosmote AE", "Greece"],
```

Comment:
The format changed to:

* [Mobile Country Code](https://en.wikipedia.org/wiki/Mobile_country_code) (MCC) (_same_)
* Mobile Network Code (_same_)
* Brand / Operator (_same_)
* Country (was Operator / Brand again)

Reply:
Yes. The initial file contained the same value on the last 2 columns (the itu document does not have the brand value, just operator name), instead of having the same value twice, was more preferable to have also the country name, this way now the country information also can be imported from the itu sources as well.